content/02.introduction.md

```diff
@@ -155,14 +155,41 @@ Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly di
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
-[@doi:10.1001/jamapediatrs.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
-Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.
-Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
-Radiologic findings were not always mentioned, but when mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
-The results of COVID-19 infection can vary greatly among pediatric patients as well as adults.
-In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].
-Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].
+#### Subpopulations of Special Concern  
+
+In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
+African Americans have also  been reported to have disproportionate risk of kidney complications from COVID-19 [@doi:10.1016/j.kint.2020.05.006].
+In addition to African Americans, disproportionate harm and mortality from COVID-19 has also been noted in other minorities, namely Latino/Hispanic communities and Native American / Alaskan Native communities such as the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055].
+In Brazil, where the pandemic has also been severe, indigenous communities are likewise at special risk [@doi:10.1126/science.abc0073].
+
+The sizable racial disparities observed may be due to a number of factors, including different levels of pre-existing co-morbidities, such as hypertension, diabetes or lung diseases.
```

Comment:
Well, I would defer to your friend, but the chronic stress explanation was supposed to be a mechanism for how being a minority hurts you, so I am not sure it would help to move it.

Reply:
@tlukan is this the line your previous comment was referring to?